functional/pipeline.py

```diff
@@ -1521,7 +1521,7 @@ def to_json(self, path, root_array=True, mode=WRITE_MODE, compression=None):
             else:
                 json.dump(self.to_dict(), output)
 
-    def to_csv(self, path, mode=WRITE_MODE, dialect='excel', compression=None, **fmtparams):
+    def to_csv(self, path, mode=WRITE_MODE, dialect='excel', compression=None, newline='\n', **fmtparams):
```

Comment:
Can you add a unit test to make sure the behavior described in the issue is fixed?

Reply:
I have added a test and made changes to the default according to the python documentation.